packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -321,6 +321,10 @@ function useMediaUploadQueue() {
 
           const trackTiming = getTimeTracker('load_upload_media');
 
+          if (resource.type === 'video' && resource.isMuted !== null) {
+            additionalData.web_stories_is_muted = resource.isMuted;
```

Comment:
`additionalData.web_stories_is_muted` might have already been set above (e.g. on line  284, when muting).

Won't this override that?

`resource` at this point is the original resource added to the queue, which means it's not yet muted. So seems like this would break things.

To prevent this, seems like this code should be moved to line ~232